src/main/java/com/locallampoon/fiveh/data/rooms.xml

```diff
@@ -61,7 +61,7 @@ busy kitchen come from a dark room, and to the SOUTH a door opens to the outside
             <exit dir="5"></exit>
         </roomExits>
         <roomItems type="array">
-            <item>Smudged Glass</item>
+            <item>smudgedGlass</item>
```

Comment:
When the user types in the command to grab an item, are they actually typing the command in camelCase? I think from a user standpoint, they should be able to type commands as they would read in English.

Reply:
closing this pull request because apparently it was a Mac issue.